service/create/service.go

```diff
@@ -492,10 +519,21 @@ func (s *Service) runMachine(input runMachineInput) error {
 			},
 		},
 	}); err != nil {
-		return microerror.MaskAny(err)
+		return nil, microerror.MaskAny(err)
 	}
 
 	s.logger.Log("info", fmt.Sprintf("instance '%s' tagged", input.name))
+	instance := reservation.Instances[0]
 
-	return nil
+	return instance, nil
```

Comment:
The interface is still changed. Did you read my comments about it? What do you think? How should we do it best?